script/Common/00_00.txt

```diff
@@ -907,8 +911,9 @@ I heard a woman's voice, clear as a bell.
 [sha:7e9b8e7482cd8cd24f727750df48740712055da7]{
 -- Page 60, Offset 281.
 -- ただでさえちっこいんだから、草むらでうずくまってると見えないのよね。気をつけなさい、あやうく蹴り飛ばされるところだったんだから」
+// 'into you' then why does shiki ask "who" later? have to be ambigious for this exchange to even make sense.
```

Comment:
I don't think this solution really makes much more sense tbqh. 
"through who" is not a logical resposne to "I almost plowed right through there"

I think the old version is fine, it just reads more indignant